componentLibraries/extensionLibrary/extensionLibrary.xml

```diff
@@ -1,26 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hopsancomponentlibrary version="0.2">
-  <!-- The "id" element gives the library a uniqe identity, a UUID is recommended but any "unique" string is accepted -->
```

Comment:
This file seems to have been changed by accident.

(It kind of annoying that these files are overwritten, but it is designed that way.
Maybe they should contain warning comments to warn people from making manual changes in them (if auto generated in the first place)) But that is an other issue.

Reply:
Yes, I mean that the file is regenerated by HopsanGenerator, and the comments are not included in the generated code, so overwritten. This is also the case for any other manual changes one has made.